drivers/comparator/Kconfig.stm32_comp

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2025 Alexander Kozhinov <ak.alexander.kozhinov@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config COMPARATOR_STM32_COMP
+	bool "ST STM32 comparator driver"
+	default y
+	depends on DT_HAS_ST_STM32_COMP_ENABLED
+	select PINCTRL
+	select USE_STM32_LL_COMP
+	select USE_STM32_LL_EXTI
+
+config COMPARATOR_STM32_COMP_EXTI_HAS_LINES_64_95
+	bool "ST STM32 EXTI has lines 64 to 95"
+	depends on DT_HAS_ST_STM32_COMP_ENABLED
+	default n
+	default y if SOC_SERIES_STM32H7X
+
+config COMPARATOR_STM32_COMP_MILLER_EFFECT_HANDLING
+	bool "ST STM32 comparator Miller effect handling"
```

Comment:
Ditto

Reply:
Changed to: "STM32 Comparator Miller effect handling"
I hope I understood you right.